products/sle/main.pm

```diff
@@ -51,7 +51,7 @@ sub remove_desktop_needles {
 }
 
 sub is_server() {
-    return get_var('FLAVOR', '') =~ /^Server/;
+    return get_var('FLAVOR', '') =~ /^(Server|SAP)/;
```

Comment:
I'm torn here. if SAP is_server I would prefer if we'd schedule it as Server-SAP-DVD. @sysrich WDYT?